### DISCUSSION TITLE
Remote server 설정하여 접속 시 에러

### DISCUSSION THREAD

**Discussion by runtostar** [naverz/zepeto-studio-kor#79] [5. Q&A]
PC 2대로 서버 접속 테스트 시, Local server는 캐릭터가 문제 없이 스폰되는데,
Remote server로 접속 시에는 아래와 같은 문제가 발생합니다. 
어떻게 해결 가능한지 확인 좀 부탁드립니다.
zepeto-multiplay-example에서 실행하였습니다.

![image](https://user-images.githubusercontent.com/95363110/145827996-bbb016c7-ef37-41f4-b2d2-cd221ff0cba4.png)

**Comment by mhkim0720** [comment]
저도 이거 안 되서 몇번 글 올렸는데 ( 1.0.6 부터 안됨. ) 된다는 리플만 달리고 해결은 안 되요
